### PR TITLE
Add alpha_dash validator.

### PR DESCRIPTION
**Pull request by DFurnes**
#### Changes

Adds the [alpha_dash](https://laravel.com/docs/5.2/validation#rule-alpha-dash) Laravel validator & tests.

---

For review: @weerd

### CODE REVIEW

File: src/rules/alpha_dash.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/**
+ * The field under validation must contain only alphanumeric characters,
+ * dashes, and underscores.
+ *
+ * @param {String} value
+ * @param {Array} params
+ * @param validate
```
Comment (weerd) [inline src/rules/alpha_dash.js]: Maybe specify where `validate()` is located? :shrug: so as not to confuse...

**Comment by weerd** [comment]
Looks good! 👍

**Comment by weerd** [comment]
Just quick comment on code docblock

File: src/rules/alpha_dash.js (hunk 1 above)
Comment (DFurnes) [inline src/rules/alpha_dash.js]: Good call. Added docblock in 405c09e.

**Pull request merged**